dnf-behave-tests/features/upgrade-from-path.feature

```diff
@@ -28,7 +28,8 @@ Scenario: Upgrade an RPM from absolute path on disk
 
 
 Scenario: Upgrade an RPM from relative path on disk
-   When I execute dnf with args "upgrade x86_64/glibc-2.28-26.fc29.x86_64.rpm x86_64/glibc-common-2.28-26.fc29.x86_64.rpm x86_64/glibc-all-langpacks-2.28-26.fc29.x86_64.rpm" from repo "dnf-ci-fedora-updates"
+  Given I set working directory to "{context.dnf.fixturesdir}/repos/dnf-ci-fedora-updates"
+   When I execute dnf with args "upgrade x86_64/glibc-2.28-26.fc29.x86_64.rpm x86_64/glibc-common-2.28-26.fc29.x86_64.rpm x86_64/glibc-all-langpacks-2.28-26.fc29.x86_64.rpm"
```

Comment:
... and another one here.

Reply:
Here is the upgrade from the relative path tested, so the working dir change is required.